test/JDBC/input/babel_varbinary.sql

```diff
@@ -58,6 +58,28 @@ go
 select 10 / 0x00;
 go
 
+--division between varbinary and int4 datatype, vice-versa
+select @@microsoftversion / 0x1000000
+go
+select 0x1000000 / @@microsoftversion
+go
+select 0x101 / 0
```

Comment:
test cases where data is coming from table columns with/without casting

Reply:
Added